ambry-store/src/test/java/com.github.ambry.store/MockId.java

```diff
@@ -52,7 +61,17 @@ public String getLongForm() {
 
   @Override
   public short sizeInBytes() {
-    return (short) (Id_Size_In_Bytes + id.length());
+    return (short) (Id_Size_In_Bytes + id.length() + Short.BYTES + Short.BYTES);
+  }
+
+  @Override
+  public short getAccountId() {
+    return accountId;
```

Comment:
I think its safer to return -1 here instead of generating random account and container IDs

Reply:
Storing random account and container ID makes the test strict. 
If code changes impact account/container validation, it helpful to find out if the change is good.